### PR TITLE
Change and document auth APIs

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: react-auth/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+See a fully working demo [here](https://github.com/reshufflehq/auth-template).
+
+### Installation
+```console
+$ npm install @reshuffle/react-auth
+```
+
+### Prerequisites
+* Install `@reshuffle/passport`
+* Configure `backend/_handler.js` to use `@reshuffle/passort`, see [instructions](../passport/README.md#Installation)
```
Comment (michaeladda) [inline react-auth/README.md]: is this link working?

Comment (arikmaor) [inline react-auth/README.md]:
```suggestion
* Configure `backend/_handler.js` to use `@reshuffle/passport`, see [instructions](../passport/README.md#Installation)
```

Comment (bergundy) [reply react-auth/README.md]: No TBD

File: react-auth/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  );
+}
+```
+
+Use with react hooks.
+
+`src/components/Main`
+```jsx
+import React from 'react';
+import { useAuth, useAuthFlow } from '@reshuffle/react-auth';
```
Comment (arikmaor) [inline react-auth/README.md]: Consider merging `useAuthFlow` into `useAuth` instead of having 2 hooks

File: react-auth/README.md (hunk 1 above)
Comment (bergundy) [reply react-auth/README.md]: Fixed

**Pull request merged**